frontend/test/unit/setup.js

```diff
@@ -21,6 +22,12 @@ config.stubs["nuxt-link"] = Vue.component("NuxtLink", {
   template: '<a :href="to" v-on="$listeners"><slot /></a>',
 })
 
+config.stubs["svg-icon"] = Vue.component("SvgIcon", {
+  props: ["name"],
+  template: '<svg title="name" />',
+})
```

Comment:
Are we able to remove the [Jest SVG transformer](https://github.com/WordPress/openverse/blob/697f62f01a32cb7fcf2f4a7627650a113cba40da/frontend/jest.config.js#L22) after this?

Reply:
We can remove the transformer after removing the other SVG module. I'd prefer to do it in another PR.